manifests/base/jupyter-trustyai-notebook-imagestream.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     # N Version of the image
     - annotations:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"TrustyAI","version":"0.3"},{"name":"Boto3","version":"1.28"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.3"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"Pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.14"}, {"name":"Sklearn-onnx","version":"1.15"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.0"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"TrustyAI","version":"0.6"},{"name":"Boto3","version":"1.28"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.3"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"Pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.14"}, {"name":"Sklearn-onnx","version":"1.15"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.0"}]'
```

Comment:
Are we okay with this change WRT this fix? I suppose we have no choice? :thinking:

Reply:
Hmmm Are you referring if this fix is worth from the Rhoai build perspective? Specifically, that this change won't be visible on the dashboard due to the Rhoai release/build procedure? Or do you have something else in mind?

Reply:
My point was more about that we're updating the TrustyAI from 0.3 to 0.6 in a supported image where we are trying to avoid as much changes as possible.

But now I realized that this PyArrow dependency is actually brought in by this package anyway, so we need to update it if we want to get rid of this CVE.

Looking on the [package release history](https://github.com/trustyai-explainability/trustyai-explainability-python/releases) - looks like we can update only to version `0.4.0` to fix the CVE in question. But probably it's not bad to move to the latest one... just saying since doing smaller bump is usually safer to avoid eventual new issues in the updated package.

Reply:
Anyway, we have this version as part of the 2023b already in other stream ([see this comment](https://issues.redhat.com/browse/RHOAIENG-4027?focusedId=24558596&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24558596)) anyway, so no need to talk about this more.

Reply:
FTR - looks like we're missing this update also in other branches, so I created opendatahub-io/notebooks!615 to start to fix that.

Reply:
Thanks for opening the PR `lgtm'd`, I added these things as notes on the PR description to be in track

Reply:
In context of this PR, only the manifest part needs to be synced to the other branches mentioned :slightly_smiling_face: